admin/app/components/solidus_admin/base_component.rb

```diff
@@ -14,7 +14,7 @@ class BaseComponent < ViewComponent::Base
     def icon_tag(name, **attrs)
       href = image_path("solidus_admin/remixicon.symbol.svg") + "#ri-#{name}"
       tag.svg(
-        class: attrs[:class],
```

Comment:
This fix is not related, right? What about using another PR, just for documentation?

Reply:
Just saw it's part of https://github.com/solidusio/solidus/pull/5268, so I guess it's here only until that one will be merged 👍

Reply:
That's it :slightly_smiling_face: